granulate_utils/java.py

```diff
@@ -215,3 +215,85 @@ def parse_jvm_version(version_string: str) -> JvmVersion:
         zing_major = None
 
     return JvmVersion(version, build, vm_name, vm_type, zing_major)
+
+
+@dataclass
+class JvmFlag:
+    name: str
+    type: str
+    value: str
+    origin: str
+    kind: List[str]
+
+    @classmethod
+    def from_match(cls, match: Match) -> "JvmFlag":
```

Comment:
You can add `from __future__ import annotations` and then no `"JvmFlags"` but `JvmFlags`.